### PR TITLE
add(translation): Bulgarian

### PR DESCRIPTION
**Pull request by Gitleptune**
Decided to contribute :), I am Bulgarian so translation should be pretty good. @Gusted how is your day going?

### CODE REVIEW

File: src/_locales/bg.config [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+#==== Store listing ====
+
+@store_listing
+Това уеб разширение с цел грижа за очите прави тъмни режими за уебсайтове в движение. Dark Reader превръща светлите цветове в тъмни, което ги прави с висок констраст и лесни за четене през нощта.
+
+Може да настройте яркостта, контраста, сепиен-филтър, тъмен режим, шрифтове и уебсайтове за игнориране.
+
+Dark Reader няма реклами и не използва вашите данни. То е с напълно отворен код https://github.com/darkreader/darkreader
+
+Преди инсталация, изключете подобни уеб разширения. Приятно гледане!
```
Comment (Gusted) [inline src/_locales/bg.config:183]: add a empty new line.

Comment (Gusted) [reply src/_locales/bg.config:183]: GitHub hide the last empty line.

**Comment by alexanderby** [comment]
@Gitleptune Hi, honestly I don't know why it's failing. Could you please start from scratch using our guideline (uploading a file directly on GitHub, not using you local computer)?

Maybe it doesn't recognise the changes in empty lines, or some other reason, e.g. CRLF endings versus LF. LF should be used.

**Comment by Gusted** [comment]
Closing.

**Pull request closed without merging**